Number_types/include/CGAL/double.h

```diff
@@ -138,8 +138,7 @@ inline double sse2fabs(double a)
   __m128d temp = _mm_set1_pd(a);
   
   temp = _mm_and_pd(temp, absMask.m);
-  _mm_store_sd(&a, temp);
-  return a;
+  return _mm_cvtsd_f64 (temp);
```

Comment:
Is that change on purpose? It does not seem related to the description of the PR?

Reply:
Marc told me that it was on purpose.